v0/destinations/firehose/transform.js

```diff
@@ -0,0 +1,53 @@
+function getDeliveryStreamMapTo(event) {
+  const config = event.destination.Config;
```

Comment:
Do we need this extra line? Could be just

`const { mapEvents } = event.destination.Config;`

Reply:
done